Cargo.toml

```diff
@@ -10,18 +10,34 @@ keywords = ["physics", "parser"]
 categories = ["science", "parser-implementations"]
 repository = "https://github.com/a-maier/hepmc2"
 
+[workspace]
+members = ["hepmc2-macros"]
+
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 log = "0.4"
 nom = "7"
 ryu = "1.0"
 strum = { version = "0.25", features = ["derive"] }
 thiserror = "1.0"
+hepmc2-macros = { version = "0.1.0", path = "hepmc2-macros" }
+tokio = { version = "1.35.1", features = [
+    "io-util",
+    "rt-multi-thread",
+], optional = true }
+maybe-async = "0.2.7"
```

Comment:
I would prefer to not pin the `tokio` and `maybe-async` patch versions.

To my understanding it generally makes most sense to either pin them for all dependencies or for none of them. There are good arguments either way, but I'd like to keep that discussion away from this pull request and stick to the current convention for the time being.

Reply:
Done